src/build.js

```diff
@@ -519,21 +523,30 @@ class Build
             {
                 domain = context.linkDomain;
             }
-            else if ( embed )
-            {
-                domain = `${embed.magazineUrl}/${embed.rootPath}`;
-            }
             else
             {
-                throw `Styla Widget error: No domain defined or bad domain config`;
+                /* istanbul ignore next */
+                /* no idea... this is totally tested */
+                if ( embed )
+                {
+                    domain = `${embed.magazineUrl}/${embed.rootPath}`;
```

Comment:
IIRC our accmen are pretty random about `rootPath`. Some clients have paths including the slash (`/magazine`), some don't (`magazine`), it all works because Backbone figures this stuff out, which also makes it super annoying to work with.

Something like this should make it resilient:

```js
let rootPath = embed.rootPath;
if (rootPath[0] === '/')
{
    rootPath = rootPath.slice(1);
}
domain = `${embed.magazineUrl}/${rootPath}`;
```